cmd/vic-machine/create/create.go

```diff
@@ -247,6 +247,13 @@ func (c *Create) Flags() []cli.Flag {
 		})
 
 	tls := c.certs.CertFlags()
+	tls = append(tls, cli.BoolFlag{
+		Name:        "no-tls",
```

Comment:
Looks like we had a short-flag `-k` for this too, should we keep it? https://github.com/vmware/vic/blob/b3f5b28d65511e5175073558d486bf397d83eb78/cmd/vic-machine/create/create.go#L301